### PR TITLE
[remote asset api] Clarify resource_type

### LINKED ISSUE #118: [remote asset api] resource_type qualifier is too vague
The remote asset api refers to `resource_type` as a recommended qualifier to resolve ambiguities, but the Qualifier Lexicon file merely has this circular definition: `resource_type: This describes the type of resource`. It's the only standard qualifier in the doc without examples- I think we should add some.

### PR DESCRIPTION
**Pull request by sstriker**
* build/bazel/remote/asset/v1/qualifiers.md
  Clarify resource_type

Closes #118.

### CODE REVIEW

File: build/bazel/remote/asset/v1/qualifiers.md [hunk 1]
```diff
@@ -5,7 +5,27 @@ This lexicon defines standard qualifier names that servers
 
 The following standard qualifier `name`s are defined:
 
-* `resource_type`: This describes the type of resource.
+* `resource_type`: This describes the type of resource.  In case of file assets use
+  an existing [media type](https://www.iana.org/assignments/media-types/media-types.xhtml).
```
Comment (mostynb) [inline build/bazel/remote/asset/v1/qualifiers.md]: Maybe reword as "File assets may use an existing media type ..." ?

File: build/bazel/remote/asset/v1/qualifiers.md [hunk 2]
```diff
@@ -5,7 +5,27 @@ This lexicon defines standard qualifier names that servers
...
+
+  It is expected to see git repositories be a common directory asset, the
+  `resource_type` to use will be `application/x-git`.
```
Comment (mostynb) [inline build/bazel/remote/asset/v1/qualifiers.md]: This reads a little strangely, how about "Git repositories should use the value  'application/x-git'"?

File: build/bazel/remote/asset/v1/qualifiers.md (hunk 1 above)
Comment (sstriker) [reply build/bazel/remote/asset/v1/qualifiers.md]: Thanks, tweaked the wording

**Comment by sstriker** [comment]
@mostynb any remaining concerns?

**Comment by mostynb** [comment]
I think this is a clear improvement- lgtm.
